variables.tf

```diff
@@ -75,6 +75,12 @@ variable "existing_cos_instance_id" {
   default     = null
 }
 
+variable "existing_cos_instance_guid" {
```

Comment:
I would recommend to take COS instance name as an input and then using a data source to get the instance id and guid

Reply:
I have adopted other code merged with master that calculates the guid from the id. This removes the need to pass guid and thus avoids this being a breaking change. This description is updated for a minor updated based on no breaking changes.